test_libs/pyspec/eth2spec/utils/ssz/test_ssz_impl.py

```diff
@@ -85,6 +85,16 @@ def merge(a: str, branch: Iterable[str]) -> str:
      "c5c2", chunk("c5c2")),
     ("bitlist TFTFFFTTFTFFFFTT", Bitlist[16](1, 0, 1, 0, 0, 0, 1, 1, 0, 1, 0, 0, 0, 0, 1, 1),
      "c5c201", h(chunk("c5c2"), chunk("10"))),
+    ("long bitvector", Bitvector[512](1 for i in range(512)),
```

Comment:
A test for bitlists/vectors of byte size, and smaller, would be nice to have.

Reply:
OK, done :)